apps/system/js/browser_context_menu.js

```diff
@@ -73,13 +73,22 @@
       // context menu api
       if (detail.contextmenu && detail.contextmenu.items.length) {
         detail.contextmenu.items.forEach(function(choice, index) {
-          items.push({
-            label: choice.label,
-            icon: choice.icon,
-            callback: function() {
-              detail.contextMenuItemSelected(choice.id);
-            }
-          });
+          var itemObj = null;
+          if (choice.id == 'copy-image') {
+            itemObj = {
+              id: choice.id,
+              label: _(choice.id),
```

Comment:
What makes `copy-image` special? If it's just about missing graphics, simply commit a temporary one is better than mess with the code like this....